drivers/lora/sx126x.c

```diff
@@ -20,15 +20,22 @@ LOG_MODULE_REGISTER(sx126x, CONFIG_LORA_LOG_LEVEL);
 #if DT_HAS_COMPAT_STATUS_OKAY(semtech_sx1261)
 #define DT_DRV_COMPAT semtech_sx1261
 #define SX126X_DEVICE_ID SX1261
+#define SX126X_GPIO
```

Comment:
What do you think about using word "external" or "standalone" or something like that instead of "GPIO"?

Reply:
Yeah fine with either. Standalone? Pick one...

Reply:
Renamed to SX126X_STANDALONE.